python/cudf/cudf/_lib/cudf.pyx

```diff
@@ -348,7 +351,7 @@ cdef Column gdf_column_to_column(gdf_column* c_col):
             nvcat_obj = nvcategory.bind_cpointer(nvcat_ptr)
             data = nvcat_obj.to_strings()
         result = as_column(data)
-        
+
     else:
         dtype = np_dtype_from_gdf_column(c_col)
         dbuf = rmm.DeviceBuffer(
```

Comment:
This is making a copy of the data for every gdf_* API. Going to build an `rmm.DevicePointer` class that manages raw pointer lifetime to handle this.

Reply:
Done